src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1248,6 +1248,12 @@
     <Compile Include="$(CommonPath)System\Runtime\CompilerServices\IsExternalInit.cs">
```

Comment:
Looks like IsExternalInit.cs, ReferenceEqualityComparer.cs and ExcludeFromCodeCoverageAttribute.cs do not follow the prevalent pattern for including public types in downlevel targets that I have commented on. You can move them under CoreLib while you are on it.